user/jwt.md

```diff
@@ -60,11 +61,12 @@ For example, using the previous configuration `SAUCE_ACCESS_KEY` and
 
 ### How secure is this addon?
 
-Very Secure. The token produced is only valid for 90 minutes, and is signed by but does not contain your secret information.
+Very Secure. The token produced is only valid for 90 minutes. It is signed in such a way to uniquely securely
+pass your secret information without worrying that information is leaked.
 
```

Comment:
What do you think about

> The JWT token is only valid for 90 minutes. It is signed in a way that lets you securely
 transmit your secret information without worrying that it is leaked.